### DISCUSSION TITLE
Subcomponents without a projenrc

### DISCUSSION THREAD

**Discussion by stuft2** [projen/projen#2505] [General]
I'm curious about the design choices that led to removing the projenrc from subcomponents. I can see pros and cons to both approaches. What was the biggest deciding factor?

Reference: https://github.com/projen/projen/releases/tag/v0.68.0

**Comment by mrgrain** [comment]
Good question! Basically the projenrc components didn't work for subprojects. 

Projenrc component tend to do three things:
- Add a `.projenrc.xyz` file\
  However rc files are only added during the initial creation with `projen new`, so this is a no-op for subprojects
 - Configure the default task (synth) e.g. making sure `python` is called with a .projenrc.py file\
  The default task errored in subprojects due to the lack of an rc-file. The linked release fixes this to call the root task instead.
- Add additional config for the rc-files, e.g. exempting `.projenrc.ts` from certain eslint rules\
  Due to the lack of an actual rc-file, these rules were just noise.

**Reply by stuft2** [reply]
Great info! Thanks!
